global.json

```diff
@@ -2,9 +2,9 @@
   "tools": {
     "dotnet": "2.1.401",
     "vs": {
-      "version": "15.8"
+      "version": "16.0"
     },
-    "xcopy-msbuild": "15.9.0-alpha"
+    "xcopy-msbuild": "16.0.0-rc1-alpha"
```

Comment:
I don't know that it'll affect anything in this repo, but just so you're aware if it does: we're taking a fix for https://github.com/Microsoft/msbuild/issues/4210 in an RC update. If you see confusing `Exec` task failures, that's probably why--holler at me and we can work around or update the MSBuild engine you're using.

Reply:
thanks for the heads up. I can update this pretty fast so once you take a fix I'll look at getting a new version.

Reply:
I just pushed 16.0.452 with the fix into `lab/d16.0stg`.